js/main.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener('DOMContentLoaded', function(){
+  let next = document.querySelector('.next');
+  next.onclick = function() {
+    document.querySelector(".container").scrollLeft += 390;
+    if (document.querySelector(".container").scrollLeft >= 900) {
+      next.classList.add('disabled');
+    }else if (document.querySelector(".container").scrollLeft >= 390) { 
+      prev.classList.remove('disabled');
+    }
+  };
+  let prev = document.querySelector('.prev');
+  prev.onclick = function() {
+    document.querySelector(".container").scrollLeft -= 390;
+    if (document.querySelector(".container").scrollLeft < 900 && document.querySelector(".container").scrollLeft > 391) {
```

Comment:
you no need this checking

Reply:
If you push button, `container.scrollLeft` changed and this `if` always be true.